scipy/optimize/optimize.py

```diff
@@ -1684,6 +1684,22 @@ def fminbound(func, x1, x2, args=(), xtol=1e-5, maxfun=500,
     interval x1 < xopt < x2 using Brent's method.  (See `brent`
     for auto-bracketing).
 
+    Examples
+    --------
+    `fminbound` finds the minimum of the function in the given range.
+    The following examples illustrate the same
+
+    >>> def f(x):
+    ...     return x**2
+    
+    >>> from scipy import optimize
+
+    >>> minimum = optimize.fminbound(f,-1,2)
```

Comment:
Yeah, there's a lot of very old code that was not written in a very nice style that we haven't changed. It's a bit of a trade-off between messing up the history by touching lots of code, and living with ``a,b`` type code. We're trying to be better about new code though.